assets/scss/6-components/full-page-loader/_full-page-loader.scss

```diff
@@ -0,0 +1,51 @@
+$loader-bar-height: 10px;
+
+/*
+  No demo for this component because, if we made one,
+  it would cover the entire page!
+
+  This component is for putting the entire width and
+  height of a page in a loading/pending state.
+
+  Inspired by:
+  https://codepen.io/brunjo/pen/XJmbNz
```

Comment:
Codepen might be a good loophole for demos like this. Maybe in our description we just link to it?

Reply:
Done.